shared/src/business/useCases/trialSessions/getCalendaredCasesForTrialSessionInteractor.test.js

```diff
@@ -55,4 +55,41 @@ describe('getCalendaredCasesForTrialSessionInteractor', () => {
       }),
     ).resolves.not.toThrow();
   });
+
+  it('should only return PMT docket entry on cases', async () => {
+    user = new User({
+      name: 'Docket Clerk',
+      role: ROLES.docketClerk,
+      userId: '6805d1ab-18d0-43ec-bafb-654e83405416',
+    });
+    applicationContext
+      .getPersistenceGateway()
+      .getCalendaredCasesForTrialSession.mockReturnValue([
+        {
+          ...MOCK_CASE,
+          docketEntries: [
+            {
+              ...MOCK_CASE.docketEntries[0],
+              eventCode: 'PMT',
+              isStricken: false,
+            },
+            {
+              ...MOCK_CASE.docketEntries[0],
+              eventCode: 'PMT',
+              isStricken: true,
+            },
+          ],
```

Comment:
The array here doesn't seem to fit with the test's name. I would have thought to see docket entries with event codes that we expect to be excluded, but instead it seems like we're checking for the stricken one to be excluded even though the event code is right?